Validation/HcalHits/interface/HcalSimHitStudy.h

```diff
@@ -48,6 +48,22 @@ class HcalSimHitStudy: public DQMEDAnalyzer {
   const HcalDDDRecConstants               *hcons;
   int                                      maxDepthHB_, maxDepthHE_;
   int                                      maxDepthHO_, maxDepthHF_;
+  int                                      maxDepth_;
+
+  struct HistLim {
+    HistLim(int nbin, double mini, double maxi)
+    : n(nbin), min(mini), max(maxi) {
+    }
+    int n;
+    double min;
+    double max;
+  };
+
+  std::map<std::string, MonitorElement*> *msm_;
```

Comment:
Hi @kencall - no need for this to be a pointer - if its preferred for some reason, it needs to be a unique_ptr. Thanks